pkg/sources/kafka/reader.go

```diff
@@ -142,20 +143,33 @@ func (r *kafkaSource) Partitions(context.Context) []int32 {
 }
 
 func (r *kafkaSource) Read(_ context.Context, count int64) ([]*isb.ReadMessage, error) {
+	latestEtMap := make(map[int32]int64)
```

Comment:
Is this for debug purpose?

Reply:
I had added it for debugging, will remove it